C06.csv

```diff
@@ -0,0 +1,161 @@
+037,Derived SI units with special names,farad,F,F,F,kg^-1 m^-2 s^4 A^2,,,Operational
+038,Derived SI units with special names,ohm,Ω,Ohm,OHM,kg m^2 s^-3 A^-2,,,Operational
+039,Derived SI units with special names,siemens,S,S,SIE,kg^-1 m^-2 s^3 A^2,,,Operational
+040,Derived SI units with special names,weber,Wb,Wb,WB,kg m^2 s^-2 A^-1,,,Operational
+041,Derived SI units with special names,tesla,T,T,T,kg s^-2 A^-1,,,Operational
+042,Derived SI units with special names,henry,H,H,H,kg m^2 s^-2 A^-2,,,Operational
+060,Derived SI units with special names,degree Celsius,˚C,Cel,CEL,K+273.15,,,Operational
+070,Derived SI units with special names,lumen,lm,lm,LM,cd sr,,,Operational
+071,Derived SI units with special names,lux,lx,lx,LX,cd sr m^-2,,,Operational
+080,Derived SI units with special names,becquerel,Bq,Bq,BQ,s^-1,,,Operational
+081,Derived SI units with special names,grey,Gy,Gy,GY,m^2 s^-2,,,Operational
+082,Derived SI units with special names,sievert,Sv,Sv,SV,m^2 s^-2,,,Operational
+110,"Other, non-SI, units recognized by CGPM",degree (angle),˚,deg,DEG,,(see Note 4),10,Operational
+111,"Other, non-SI, units recognized by CGPM",minute (angle),',',MNT,,,,Operational
+112,"Other, non-SI, units recognized by CGPM",second (angle),'','',SEC,,,,Operational
+120,"Other, non-SI, units recognized by CGPM",litre,l or L,l or L,L,,,,Operational
+130,"Other, non-SI, units recognized by CGPM",minute (time),min,min,MIN,,,,Operational
+131,"Other, non-SI, units recognized by CGPM",hour,h,h,HR,,,,Operational
+132,"Other, non-SI, units recognized by CGPM",day,d,d,D,,,,Operational
+150,"Other, non-SI, units recognized by CGPM",tonne,t,t,TNE,,,,Operational
+160,"Other, non-SI, units recognized by CGPM",electron volt,eV,eV,EV,,,,Operational
+161,"Other, non-SI, units recognized by CGPM",atomic mass unit,u,u,U,,,,Operational
+170,"Other, non-SI, units recognized by CGPM",astronomic unit,AU,AU,ASU,,,,Operational
+171,"Other, non-SI, units recognized by CGPM",parsec,pc,pc,PRS,,,,Operational
+200,Non-SI Units tolerated because of widespread use,nautical mile, ,,,,(see Note 7),21,Operational
+201,Non-SI Units tolerated because of widespread use,knot,kt,kt,KT,,,,Operational
+210,Non-SI Units tolerated because of widespread use,decibel,dB,dB,DB,,(see Note 6),17,Operational
+220,Non-SI Units tolerated because of widespread use,hectare,ha,ha,HAR,,,,Operational
+230,Non-SI Units tolerated because of widespread use,week, ,,,,,,Operational
+231,Non-SI Units tolerated because of widespread use,year,a,a,A,,,,Operational
+300,Other Units as used by WMO,per cent,%,%,PERCENT,,,,Operational
+301,Other Units as used by WMO,parts per thousand,‰,0/00,PERTHOU,,,,Operational
+310,Other Units as used by WMO,eighths of cloud,okta,okta,OKTA,,,,Operational
+320,Other Units as used by WMO,degrees true,˚,deg,DEG,,,,Operational
+321,Other Units as used by WMO,degrees per second,degree/s,deg/s,DEG/S,,,,Operational
+350,Other Units as used by WMO,degrees Celsius,˚ C,C,C,,(see Note 8),13,Operational
```

Comment:
Delete the space between the degree symbol and the C, they are one unit. With the space we have degree Coulomb.

Reply:
this is also applicable to other entries below.